packages/cli/src/commands/devHandler.js

```diff
@@ -49,8 +51,78 @@ export const handler = async ({
   }
 
   if (side.includes('web')) {
+    let proposedPort = parseInt(getConfig().web.port)
```

Comment:
Just checked and looks like you don't have to parse it:

```suggestion
    let proposedPort = getConfig().web.port
```

Reply:
Okay this parsing was added because I, under some condition I can't quite recall right now, encountered an issue without it. I'll do some more testing without it and if it turns out it's definitely not needed I'll remove it or I'll report back with the reason for needing it.

It may have been one of the other port checking libraries I tested needed specifically an integer passed into it. I'll check double check portfinder.

Reply:
This `parseInt` related issue has been resolved in the latest commits. It's only used once now and it is needed.